src/index.html

```diff
@@ -1,17 +1,182 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+     action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+     method="post"
+     class="form">
+      <fieldset class="fieldset">
+
+        <legend>Personal information</legend>
+
+        <label for="sname" class="form-field">
+          Surname:
+          <input
+           type="text"
+           name="surname"
+           autocomplete="off"
+           minlength="1"
+           maxlength="40"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
            minlength="1"
            maxlength="40"
          >
```
3 point in checklist, you need to add 2 spaces indentation, now you only have one, make sure you fix it everywhere not just here